source/tests/test_nvnmd_entrypoints.py

```diff
@@ -32,22 +33,30 @@ def test_freeze(self):
             "layer_0_type_0/bias",
             "final_layer_type_0/matrix",
             "final_layer_type_0/bias",
-        )
+        ]
+        namelist2 = [
+            "train_attr/min_nbor_dist"
+        ]
+        namelist = namelist1 + namelist2
+        # crete variable as namelist
         tvlist = []
-        save_path = str(tests_path / os.path.join("nvnmd", "weight.npy"))
+        save_path = str(tests_path / os.path.join("nvnmd/out", "weight.npy"))
```

Comment:
```suggestion
        save_path = str(tests_path / "nvnmd" / "out" / "weight.npy")
```

Reply:
Thank you very much. We have made changes.